Artesian/Artesian.SDK/Dto/GMEPublicOffer/GMEPublicOfferCurveDto.cs

```diff
@@ -13,19 +13,19 @@ public class GMEPublicOfferCurveDto
         /// The Status
         /// </summary>
         [Key(0)]
-        public Status Status { get; set; }
+        public Status? Status { get; set; }
 
         /// <summary>
         /// The BAType
         /// </summary>
         [Key(1)]
-        public BAType BAType { get; set; }
+        public BAType? BAType { get; set; }
```

Comment:
enums are value type. do not change nullability. add ctor or migrate to immutable record with init only.

apply this everywhere to DTOs